Mage.Sets/src/mage/cards/n/NeyamShaiMurad.java

```diff
@@ -0,0 +1,129 @@
+package mage.cards.n;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.effects.OneShotEffect;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.constants.Zone;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.game.events.DamagedEvent;
+import mage.game.events.GameEvent;
+import mage.players.Player;
+import mage.target.common.TargetCardInGraveyard;
+import mage.target.targetpointer.FixedTarget;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.cards.Cards;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+
+/**
+ *
+ * @author Xanderhall
+ */
+public final class NeyamShaiMurad extends CardImpl {
+
+    public NeyamShaiMurad(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{2}{W}{B}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.ROGUE);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Rogue Trader -- Whenever Neyam Shai Murad deals combat damage to a player, you may have that player return target permanent card from their graveyard to their hand.
+        // If you do, that player chooses a permanent card in your graveyard, then you put it onto the battlefield under your control.
+
+        this.addAbility(new NeyamShaiMuradTriggeredAbility().withFlavorWord("Rogue Trader"));
+    }
+
+    private NeyamShaiMurad(final NeyamShaiMurad card) {
+        super(card);
+    }
+
+    @Override
+    public NeyamShaiMurad copy() {
+        return new NeyamShaiMurad(this);
+    }
+}
+
+class NeyamShaiMuradTriggeredAbility extends TriggeredAbilityImpl {
```

Comment:
why not use `DealsCombatDamageToAPlayerTriggeredAbility`?

Reply:
No idea! Probably a remnant of different iterations of trying to get the effect to work properly. |3